src/virtual_packages.cpp

```diff
@@ -92,7 +92,50 @@ namespace mamba
 
             if (ec)
             {
-                LOG_ERROR << "Could not find CUDA version by calling 'nvidia-smi' (skipped)\n";
+                out = "";
+#if defined(_WIN32)
+                bool may_exist = false;
+                std::string path = env::get("PATH");
+                std::vector<std::string> paths = split(path, env::pathsep());
+
+                for (auto p : paths)
+                {
+                    if (fs::exists(concat(p, env::pathsep(), "nvcuda.dll")))
+                    {
+                        may_exist = true;
+                        break;
+                    }
+                }
+
+                std::string base = "C:\\Windows\\System32\\DriverStore\\FileRepository";
+                if (may_exist)
+                {
+                    for (fs::path p : fs::directory_iterator(base))
+                    {
+                        if (p.string().startswith("nv")
+                            && fs::exists(concat(p, env::pathsep(), "nvidia-smi.exe")))
+                        {
+                            std::string err;
+                            std::string f = fs::exists(concat(p, env::pathsep(), "nvidia-smi.exe"));
```

Comment:
? looks liek a bug (assigning bool to string)

Reply:
it is !